src/Gui/ClearableButtonGroup.cpp

```diff
@@ -0,0 +1,53 @@
+ /**************************************************************************
+ *   Copyright (c) 2021 FreeCAD Developers                                 *
+ *   Author: Ajinkya Dahale                                                *
```

Comment:
please add a mail address - all other devs have this since copyright holders must be reachable somehow